internal/twirptest/client_test.go

```diff
@@ -299,3 +326,12 @@ func errCause(err error) error {
 	}
 	return cause
 }
+
+// wrappedHTTPClient is a http.Client which is wrapped to test against the HTTPClient interface
```

Comment:
This really should be changed to good english ...

Reply:
Eh, it's not too bad. It's just an unexported type in test code. Maybe something like:
```go
// wrappedHTTPClient implements HTTPClient, but can be inspected during tests.
```